src/subsystem/drive_controller.cpp

```diff
@@ -71,3 +71,31 @@ void DriveController::set_drive_mode(DriveMode mode) {
 		update_motor_acceleration();
 	}
 }
+
+//Initialize the drive (CAN bus initialization commands)
+bool DriveController::drive_init() {
+	for (int i = static_cast<int>(Node::DRIVE_AXIS_0); i <= static_cast<int>(Node::DRIVE_AXIS_5); i++) {
+#ifdef ONBOARD_CAN_BUS
```

Comment:
I don't think we should use these macros in the drive controller. Is there anything wrong with calling these methods regardless of whether CAN is in onboard/offline mode?